content/id/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -0,0 +1,303 @@
+
+    Secara bawaan, sebuah klaster Kubernetes akan membuat namespace bawaan ketika menyediakan klaster untuk menampung Pod, Services, dan Deployment yang digunakan oleh klaster.
+    
+    Dengan asumsi kamu memiliki klaster baru, kamu bisa mengecek namespace yang tersedia dengan melakukan hal berikut:
+
+    ```shell
+    kubectl get namespaces
+    ```
+    ```
+    NAME      STATUS    AGE
+    default   Active    13m
+    ```
+
+2. Membuat namespace baru
+
+	 Untuk latihan ini, kita akan membuat dua namespace Kubernetes tambahan untuk menyimpan konten kita
+
+    Dalam sebuah skenario dimana sebuah organisasi menggunakan klaster Kuberetes yang digunakan bersama untuk pengguaan _development_ dan _production_:
+
+    Tim pengembang ingin mengelola ruang di dalam klaster dimana mereka bisa melihat daftar Pod, Layanan, dan Deployment yang digunakan untuk membangun dan menjalankan apliksi mereka. Di ruang ini sumber daya akan datang dan prgi dan pembatasan mengenai siapa bisa atau tidak bisa memodifikasi sumber daya lebih santai untuk mendukung pengembangan secara _agile_.
+    
+    Tim operasi ingin mengelola ruang didalam klaster dimana mereka bisa memaksakan prosedur ketat mengenai siapa yang bisa atau tidak bisa melakukan manipulasi kumpulan Pod, Layanan, dan Deployment yang berjalan di situs _production_.
+
+    Satu pola yang bisa diikuti organisasi ini adalah dengan membagi klaster Kubernetes menjadi dua namespace: `development` dan `production`
+
+    Mari kita buat dua namespace untuk menyimpan hasil kerja kita.
+
+    Buat namespace `development` menggunakan kubectl:
+
+    ```shell
+    kubectl create -f https://k8s.io/examples/admin/namespace-dev.json
+    ```
+
+    Kemudian mari kita buat namespace `production` menggunakan kubectl:
+
+    ```shell
+    kubectl create -f https://k8s.io/examples/admin/namespace-prod.json
+    ```
+
+    Untuk memastikan apa yang kita lakukan benar, lihat seluruh namespace didalam klaster.
+
+    ```shell
+    kubectl get namespaces --show-labels
+    ```
+    ```
+    NAME          STATUS    AGE       LABELS
+    default       Active    32m       <none>
+    development   Active    29s       name=development
+    production    Active    23s       name=production
+    ```
+
+3. Buat pod di tiap namespace
+
+    Sebuah namespace Kubernetes memberikan batasan untuk Pods, Layanan, dan Deployment di dalam klaster.
+
+    Pengguna yang berinteraksi dengan salah satu namespace tidak melihat konten di dalam namespace lain
+    
+    Untuk menunjukkan hal ini, Mari kita jalankan Deployment dan Pods sederhana di dalam namespace `development`.
+
+    ```shell
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname -n=development
+    kubectl scale deployment snowflake --replicas=2 -n=development
+    ```
+    Kita baru aja membuat sebuah deployment yang memiliki ukuran replika 2 menjalankan pod dengan nama `snowflake` dengan sebuah kontainer dasar yang hanya melayani hostname.
```

Comment:
Pod, Container and Deployment are API name, use capital.
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```suggestion
    Kita baru aja membuat sebuah Deployment yang memiliki ukuran replika dua yang menjalankan Pod dengan nama `snowflake` dengan sebuah Container dasar yang hanya melayani _hostname_.
```